docs/resources/projects/settings/project-setup.md

```diff
@@ -0,0 +1,535 @@
+---
+slug: project-setup
+title: Project Setup
+tags: [permissions, multiple languages, multi-platform, walkthrough]
```

Comment:
Same comment about tags starting with capital letter. And I know we use ChatGPT for it, but we should carefully filter the tags in each page. Each time we add a tag here, Docusaurus will create a new page called ../tags/permissions so all pages tagged permissions will appear there. So we need to make sure we dont create unnecessary tags. Please see all the pages in this PR and ensure we only create relevant tags that can be reused for diffrent pages (example "setup" can be reused between flutterflow setup, app setup, project setup)

Reply:
Yes, got it!